lib/CleantalkSP/SpbctWP/Firewall/UploadChecker.php

```diff
@@ -301,6 +306,25 @@ public static function runCheckForWordpressModules(
         // Output result message
         if ( $overall_result ) {
             if (self::checkVulnerability($source)) {
+                echo ('<b>' . __('If you want to continue the installation, add the file and click install.', 'security-malware-firewall') . '</b>');
```

Comment:
It is a good practice to echoing with `echo Escape::ksesPreset('htmlcode,'preset')`. We have received the Wordpress team warning for unescaped echo stuff in the past.

Reply:
Escape::escKsesPreset is correct method.